Application Settings/StudentRegestrationPage.page

```diff
@@ -0,0 +1,32 @@
+<apex:page standardController="Student__c" extensions="StudentRegestrationController"
```

Comment:
1. Picklist values are not getting translated
2. Student Name label HSC SCC labels not getting translated
3. Please add a label "Select Language" before the language picklist.
4. I want loading status whenever we select the language.